FlowCal/io.py

```diff
@@ -1243,36 +1258,68 @@ def resolution(self, channels=None):
         else:
             return self._resolution[channels]
 
-    def hist_bins(self, channels=None, nbins=None, scale='linear'):
+    def hist_bins(self, channels=None, nbins=None, scale='linear', **kwargs):
```

Comment:
gonna bump this, in light of all the discussion. should the default here for `hist_bins()` be `logicle`? or do you still think it should be `linear`, in spite of the embrace of logicle everywhere else? not sure if this snuck through un-updated or if this was intentionally left linear.